packages/shylu/shylu_dd/frosch/src/InterfaceSets/FROSch_DDInterface_def.hpp

```diff
@@ -473,8 +472,8 @@ namespace FROSch {
                 tmpNodesMap = Xpetra::MapFactory<LO,GO,NO>::Build(NodesMap_->lib(),-1,indices(),0,NodesMap_->getComm());
             }
 
-            Teuchos::RCP<Xpetra::Matrix<SC,LO,GO,NO> > commMat = Xpetra::MatrixFactory<SC,LO,GO,NO>::Build(tmpNodesMap,10);
-            Teuchos::RCP<Xpetra::Matrix<SC,LO,GO,NO> > commMatTmp = Xpetra::MatrixFactory<SC,LO,GO,NO>::Build(tmpUniqueNodesMap,10);
+            GraphPtr commGraph = Xpetra::CrsGraphFactory<LO,GO,NO>::Build(tmpNodesMap,10);
+            GraphPtr commGraphTmp = Xpetra::CrsGraphFactory<LO,GO,NO>::Build(tmpUniqueNodesMap,10);
 #ifdef Tpetra_issue_1752
             // AH 10/10/2017: Can we get away with using just one importer/exporter after the Tpetra issue is fixed?
```

Comment:
We fixed that issue in 2017.

Reply:
You are right, I never removed this. I did this now.